package.json

```diff
@@ -31,7 +32,8 @@
 		"eslint": "8.57.1",
 		"husky": "8.0.3",
 		"jetpack-cli": "workspace:*",
-		"jetpack-js-tools": "workspace:*"
+		"jetpack-js-tools": "workspace:*",
+		"react-scanner": "1.2.0"
```

Comment:
We've been trying to keep the dependencies of the monorepo root minimal. Can this be put into `tools/js-tools/package.json` instead, possibly with a stub executable like we do for jest, prettier, and so on?